Example/Backpack/SceneDelegate.swift

```diff
@@ -0,0 +1,87 @@
+//
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import Backpack
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+    var isUITestingEnabled = false
+
+    // swiftlint:disable line_length
```

Comment:
Any reason to disable the lint rule?

Reply:
When I split it into multiple lines, it will complain about indentation :/.

Reply:
Can the indentation be fixed? Or are we using an incompatible ruleset?

Reply:
With the current ruleset we either:
- hit `line_length`
- hit `indentation_width`
But when we solve indentation_width, we will hit `vertical_parameter_alignment`

@frugoman What's your opinion on this?
We could disable the `line_length` or `indentation_width`, both would be fair game. But we can't get around both of these at the same time :( 

Relevant discussion:
https://github.com/realm/SwiftLint/issues/3046

Reply:
In these cases I do commonly prefer this notation
```swift
func scene(
   _ scene: UIScene,
   willConnectTo session: UIScene,
   ....
) {
   //code here
}
```
not sure if there's a rule against it tho